e2e-tests/IIoTPlatform-E2E-Tests/deploy/IoTHubEdgeBaseDeployment.cs

```diff
@@ -0,0 +1,116 @@
+                                ""loggingOptions"": """",
+                                ""registryCredentials"": {
+                                }
+                            }
+                        },
+                        ""systemModules"": {
+                            ""edgeAgent"": {
+                                ""type"": ""docker"",
+                                ""settings"": {
+                                    ""image"": ""mcr.microsoft.com/azureiotedge-agent:" + version+ @""",
+                                    ""createOptions"": ""{}""
+                                },
+                                ""env"": {
+                                    ""ExperimentalFeatures__Enabled"": {
+                                        ""value"": ""true""
+                                    },
+                                    ""ExperimentalFeatures__EnableGetLogs"": {
+                                        ""value"": ""true""
+                                    },
+                                    ""ExperimentalFeatures__EnableUploadLogs"": {
+                                        ""value"": ""true""
+                                    },
+                                    ""ExperimentalFeatures__EnableMetrics"": {
+                                        ""value"": ""true""
+                                    }
+                                }
+                            },
+                            ""edgeHub"": {
+                                ""type"": ""docker"",
+                                ""status"": ""running"",
+                                ""restartPolicy"": ""always"",
+                                ""settings"": {
+                                    ""image"": ""mcr.microsoft.com/azureiotedge-hub:" + version + @""",
+                                    ""createOptions"":  ""{\""HostConfig\"":{\""PortBindings\"":{\""443/tcp\"":[{\""HostPort\"":\""443\""}],\""5671/tcp\"":[{\""HostPort\"":\""5671\""}],\""8883/tcp\"":[{\""HostPort\"":\""8883\""}],\""9600/tcp\"":[{\""HostPort\"":\""9600\""}]}},\""ExposedPorts\"":{\""5671/tcp\"":{},\""8883/tcp\"":{},\""9600/tcp\"":{}}}""
+                                }
+                            }
+                        },
+                        ""modules"": {
+                        }
+                    }
+                },
+                ""$edgeHub"": {
+                    ""properties.desired"": {
+                        ""schemaVersion"": """ + kDefaultSchemaVersion + @""",
+                        ""storeAndForwardConfiguration"": {
+                            ""timeToLiveSecs"": 7200
+                        }
+                    }
+                }
+            }
+            ");
+        }
+
+        private const string kDefaultSchemaVersion = "1.0";
+        private const string kDeploymentName = "iiotedge";
+        private readonly IIoTPlatformTestContext _context;
```

Comment:
You do not need `_context` in `IoTHubEdgeBaseDeployment`. Please remove it as also base already has it.

Reply:
Removed and changed access from base class to protected